app/models/concerns/thredded/user_topic_read_state_common.rb

```diff
@@ -9,7 +9,7 @@ module UserTopicReadStateCommon
 
     # @return [Boolean]
     def read?
-      postable.updated_at <= read_at
+      postable.updated_at.to_i <= read_at.to_i
```

Comment:
It must, then, set it down to the second without any specific sub-second
accuracy because the database will save it with nanosecond accuracy. It's
really weird - I completely agree

On Saturday, April 23, 2016, Gleb Mazovetskiy <notifications@github.com>
wrote:

> In app/models/concerns/thredded/user_topic_read_state_common.rb
> <https://github.com/thredded/thredded/pull/258#discussion_r60825741>:
>
> > @@ -9,7 +9,7 @@ module UserTopicReadStateCommon
> >
> >      # @return [Boolean]
> >      def read?
> > -      postable.updated_at <= read_at
> > +      postable.updated_at.to_i <= read_at.to_i
>
> @jayroh <https://github.com/jayroh> This is strange, the times should be
> identical, because read_at is set to updated_at of the last post.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/thredded/thredded/pull/258/files/2c85076ba1034a3664abc887f89b22e67405564d#r60825741>
>